graphql/src/main/java/io/micronaut/configuration/graphql/ws/apollo/GraphQLApolloWsResponse.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.micronaut.configuration.graphql.apollo.ws;
+package io.micronaut.configuration.graphql.ws.apollo;
```

Comment:
don't change the package unless we need to that could be an unnecessary breaking change

Reply:
I don't believe the `apollo.ws` package has ever been released in anything other than the 4.0 milestones. Making `apollo` a subpackage of `ws` makes much more logical sense to me.